docs/user-guide/deploying/release-management.md

```diff
@@ -18,7 +18,7 @@ Besides the version tags you can also use the "latest" and "develop" tags which
 
 ## Custom Projects
 
-When using a [custom deployment](custom-deployment) you need to specify what OpenRemote version it customizes.
+When using a [custom deployment](https://docs.openremote.io/docs/user-guide/deploying/custom-deployment/) you need to specify what OpenRemote version it customizes.
```

Comment:
What is the rationale for changing the link? It seems to work fine.

It's used like this so Docusaurus can use relative paths and check that files/paths exist. This way the links work properly if you run the documentatation website locally or on another host. Versioned docs also won't work if you use hard coded URLs like this. Your link points to the latest release instead of the release you've selected in the dropdown.

Reply:
If the issue is that the links don't work on GitHub. Maybe we can fix that by adding using `custom-deployment.md` as link target.